test/sinatra/authorization_helpers_test.rb

```diff
@@ -0,0 +1,40 @@
+require 'test_helper'
+require 'json'
+require 'sinatra/base'
+
+ENV['RACK_ENV'] = 'test'
+
+class AuthorizationHelpersTest < Test::Unit::TestCase
+  include Rack::Test::Methods
+
+  def app
+    TestApp.new
+  end
+
+  def test_http
+    get '/public'
+    assert last_response.ok?
+    get '/private'
+    assert last_response.ok?
```

Comment:
Having tested this: this relies on `localhost` being in the `trusted_hosts`. On a HTTP call it uses reverse DNS + forward DNS. This isn't very secure but since the default deployment only enabled modules on HTTP when they are designed to assume no security it's not that big of a deal.

Reply:
Yes, this is how it have been done for a while: using authenticated http is not very secure and people should count on it. Anyway, not trying to address it here

Reply:
I do think it would be good to capture this in a test. Perhaps set the trusted hosts to empty or something different and assert that the response it not ok? Could be separate tests.

Reply:
The subject of this test is ensuring the authorization helpers are available. I'm not touching the authorization algorigth itself, and the tests for it are somewhere else: I would prefer keeping this PR focused on thing only: which is making the methods available outside of the before filters, that right now we are constraint with.

Reply:
Agreed, more tests is good but I want this to be merged ASAP, let's improve somewhere else.

Reply:
My argument was that if the response is ok, you still don't really know if the authorization was properly called.